webapp/src/app.tsx

```diff
@@ -3896,37 +3898,74 @@ export class ProjectView
                         <notification.NotificationBanner parent={this} />
                         <container.MainMenu parent={this} />
                     </header>}
-                {inTutorial && <div id="maineditor" className={sandbox ? "sandbox" : ""} role="main">
-                    <tutorial.TutorialCard ref={ProjectView.tutorialCardId} parent={this} pokeUser={this.state.pokeUserComponent == ProjectView.tutorialCardId} />
-                    {flyoutOnly && <tutorial.WorkspaceHeader />}
-                </div>}
-                <div id="simulator" className="simulator">
-                    <div id="filelist" className="ui items">
-                        <div id="boardview" className={`ui vertical editorFloat`} role="region" aria-label={lf("Simulator")} tabIndex={inHome ? -1 : 0}>
+                {sidebarTutorial?
+                    <div>
+                    {inTutorial && <div id="maineditor" className={sandbox ? "sandbox" : ""} role="main">
```

Comment:
i think you only need to switch out the tutorial.TutorialCard component, you can leave the simulator and main editor components as-is--those only require CSS changes